services/gateway/apis/http-exports/swagger/parameters/export.json

```
@@ -25,5 +25,12 @@
         "format": "interval",
         "minLength": 10,
         "maxLength": 21
+    },
+    "filename": {
+        "name": "filename",
+        "in": "path",
+        "description": "Filename",
+        "required": true,
+        "type": "string"
```

The length of the `filename` would always be `80 characters` long since we follow the following naming convention: `transactions_${address}_${from}_${to}.csv`, where `address` is `41` characters long and both `from` & `to` are `10` characters each (`YYYY-MM-DD` format).

```suggestion
        "type": "string",
        "minLength": 80,
        "maxLength": 80
```